CHANGELOG.md

```diff
@@ -6,6 +6,16 @@
 # 2.66.x
 
 # 2.65.x
+* Update Razor to 9.0.0-preview.25113.1 (PR: [#7989](https://github.com/dotnet/vscode-csharp/pull/7989))
```

Comment:
this needs to move to the next section

Reply:
2.66.x? Or a new section?

Reply:
2.66 - the changelog versions are autogenerated now, so the top most one should always be correct for whatever branch you're in